imaging-ingestion-operator/model/common_test.go

```diff
@@ -0,0 +1,203 @@
+/*
+(C) Copyright IBM Corp. 2021
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package model_test
```

Comment:
Please use same package name for the test also so that we are consistent with the usage in other places. I don't think we need a different package name for unit tests.

Reply:
done